app/graphql/types/query_type.rb

```diff
@@ -33,5 +35,27 @@ def markets_by_location(lat:, lng:, radius:, products: '')
         location: location
       }
     end
+
+    # Return markets by city, state
+    field :markets_by_city, Types::ReturnType, null: false do
```

Comment:
cool looks good that's basically the other one but using the city instead, thanks for getting that one in